src/routes/manage-selections/ManageSelectionsScreen.tsx

```diff
@@ -24,11 +24,17 @@ const Root = styled.ScrollView`
 
 const SectionHeading = styled(Subheading)`
   padding-top: ${props => props.theme.spacings.xl};
+  font-family: ${props => props.theme.fonts.contentFontBold};
+  color: ${props => props.theme.colors.text};
```

Comment:
This component should inherit from the SubheadingText component, please replace the "Subheading" with "SubheadingText" and adjust the import (previously Subheading was imported from the wrong source). Afterwards you can remove `font-familiy` and `color` from this component.